index.js

```diff
@@ -0,0 +1,47 @@
+const config = {
+  "env": {
+    "browser": true,
+    "es6": true,
+    "mocha": true,
+    "node": true
```

Comment:
Is it fine if some of these are mutually exclusive? i.e. `"browser"` and `"node"` will never be true at the same time, right?